stacks/vitess/README.md

```diff
@@ -0,0 +1,78 @@
+# Getting Started with the Vitess Operator
+
+After you have deployed this stack you will have the Vitess operator and an
+example Database cluster. 
+
+## Base Requirements
+The example database requires four persistent volume claims ( 3 for the
+lockserver, and 1 for the database ). It also requires three nodes to function
+effectively. The operator will make its best effort to spread all components
+across nodes to ensure availability. 
+
+## Accessing the Cluster
+To log into this cluster you can use the following commands
+
+
+### Access the Database from outside the Kubernetes cluster
+You can access the database outside the kubernetes cluster by port-forwarding to
+the vtgate service. This service will load balance connection to all vtgates.
+
+``` sh
+VTGATE=$(kubectl get service -l "planetscale.com/component=vtgate,planetscale.com/cluster=example" -o custom-columns=Name:.metadata.name | tail -n -1)
+kubectl port-forward service/"${VTGATE}" 3306&
+```
+
+you will then be able to access your vitess cluster on the forwarded port with
+the standard mysql client 
+
+``` sh
+mysql -h 127.0.0.1 -u admin -pscalewithvitess main 
+```
+
+### Access the Database from inside the Kubernetes cluster
+Inside the kubernetes cluster the vtgate service will be accessable on the
```

Comment:
Suggestions: "Inside the Kubernetes cluster, the vtgate service will be accessible on the
service DNS name. You can find all vtgate services using the following command:"